models/public/index.md

```diff
@@ -9,64 +9,64 @@ The models can be downloaded via Model Downloader
+| HBONet 0.25                 | PyTorch\*                          | [hbonet-0.25](./hbonet-0.25/hbonet-0.25.md) | 57.3/79.8 | 0.0758 | 1.9299 |
+| Inception (GoogleNet) V1    | Caffe\*<br>TensorFlow\*            | [googlenet-v1](./googlenet-v1/googlenet-v1.md)<br>[googlenet-v1-tf](./googlenet-v1-tf/googlenet-v1-tf.md) | 68.928/89.144<br>69.814/89.6 | 3.016~3.266 | 6.619~6.999 |
+| Inception (GoogleNet) V2    | Caffe\*<br>TensorFlow\*            | [googlenet-v2](./googlenet-v2/googlenet-v2.md)<br>[googlenet-v2-tf](./googlenet-v2-tf/googlenet-v2-tf.md) | 72.024/90.844<br>74.084/91.798 | 4.058 | 11.185 |
+| Inception (GoogleNet) V3    | TensorFlow\*<br>PyTorch\*          | [googlenet-v3](./googlenet-v3/googlenet-v3.md) <br> [googlenet-v3-pytorch](./googlenet-v3-pytorch/googlenet-v3-pytorch.md) | 77.904/93.808<br>77.696/93.696 | 11.469 | 23.817 |
+| Inception (GoogleNet) V4    | TensorFlow\*                       | [googlenet-v4-tf](./googlenet-v4-tf/googlenet-v4-tf.md) | 80.204/95.21 | 24.584 | 42.648 |
+| Inception-ResNet V2         | TensorFlow\*                       | [inception-resnet-v2-tf](./inception-resnet-v2-tf/inception-resnet-v2-tf.md) | 80.14/95.10 | 22.227 | 30.223 |
+| MobileNet V1 0.25 128       | Caffe\*                            | [mobilenet-v1-0.25-128](./mobilenet-v1-0.25-128/mobilenet-v1-0.25-128.md)  | 40.54/65 | 0.028 | 0.468 |
+| MobileNet V1 0.5 160        | Caffe\*                            | [mobilenet-v1-0.50-160](./mobilenet-v1-0.50-160/mobilenet-v1-0.50-160.md) | 59.86/82.04 | 0.156 | 1.327 |
+| MobileNet V1 0.5 224        | Caffe\*                            | [mobilenet-v1-0.50-224](./mobilenet-v1-0.50-224/mobilenet-v1-0.50-224.md) | 63.042/84.934| 0.304 | 1.327 |
+| MobileNet V1 1.0 224        | Caffe\*<br>TensorFlow\*            | [mobilenet-v1-1.0-224](./mobilenet-v1-1.0-224/mobilenet-v1-1.0-224.md)<br>[mobilenet-v1-1.0-224-tf](./mobilenet-v1-1.0-224-tf/mobilenet-v1-1.0-224-tf.md)| 69.496/89.224<br>71.03/89.94 | 1.148 | 4.221 |
+| MobileNet V2 1.0 224        | Caffe\*<br>TensorFlow\*<br>PyTorch\*| [mobilenet-v2](./mobilenet-v2/mobilenet-v2.md) <br>[mobilenet-v2-1.0-224](./mobilenet-v2-1.0-224/mobilenet-v2-1.0-224.md)<br>[mobilenet-v2-pytorch](./mobilenet-v2-pytorch/mobilenet-v2-pytorch.md) | 71.218/90.178<br>70.20/89.61<br>71.81/90.396 | 0.615~0.876 | 3.489 |
+| MobileNet V2 1.4 224        | TensorFlow\*                       | [mobilenet-v2-1.4-224](./mobilenet-v2-1.4-224/mobilenet-v2-1.4-224.md) | 74.09/91.97 | 1.183  | 6.087 |
+| MobileNet V3 Small 1.0      | TensorFlow\*                       | [mobilenet-v3-small-1.0-224-tf](./mobilenet-v3-small-1.0-224-tf/mobilenet-v3-small-1.0-224-tf.md) | 67.36/87.45 | 0.121 | 2.537 |
+| MobileNet V3 Large 1.0      | TensorFlow\*                       | [mobilenet-v3-large-1.0-224-tf](./mobilenet-v3-large-1.0-224-tf/mobilenet-v3-large-1.0-224-tf.md) | 75.70/92.76 | 0.4536 | 5.4721 |
```

Comment:
not sure, but possible open-closed-eye model should be in this list as binary classification model

Reply:
Added